packages/environment-check/src/checks/service-checks.ts

```diff
@@ -245,8 +246,16 @@ export async function checkTransportRequests(
             logger.warn(t('warning.getTransportRequestsoNotAvailable'));
         }
     } catch (e) {
+        logger.debug('error: ' + JSON.stringify(e));
         logger.error(t('error.getTransportRequestsError'));
-        logger.debug(e.message);
+
+        if (e.message.includes('403')) {
+            {
+                logger.debug(e.message + '\n' + t('warning.transportRequests403'));
+            }
+        } else {
+            logger.debug(e.message);
```

Comment:
Not sure I understand what the aim is here. You've already logged the error `JSON.stringify(e)` which would include the message as part of the error object. 

Maybe line `logger.debug('error: ' + JSON.stringify(e));` can be removed?

Reply:
Also, this is called `warning.transportRequests403` when its being logged as `debug`, I would expect this to be `logger.warn`

Reply:
@longieirl I see what you mean. I wanted to add the link text and link to the debug section, under the warning if it makes any sense?

Reply:
Changed as requested.